contributing_to_docs/term_glossary.adoc

```diff
@@ -203,6 +203,26 @@ Open the [filename]#Dockerfile# and make the following changes.
 Create a [filename]#Dockerfile# at the root of your repository.
 ====
 
+== E
+
+''''
+=== event
+
+Usage: event(s)
+
+An event is a data record expressing an occurrence and its context, based on the CNCF CloudEvents specification.
+Events contains two types of information: the event data representing the occurrence, and the context metadata providing contextual information about the occurrence.
+Events are routed from an event producer, or source, to connected event consumers.
+
+Routing can be performed based on information contained in the event, but an event will not identify a specific routing destination.
+Events can be delivered through various industry standard protocols such as HTTP, AMQP, MQTT, or SMTP, or through messaging and broker systems such as Kafka, NATS, AWS Kinesis, or Azure Event Grid.
+
+While Events are a Kubernetes Metadata API object, this term is not commonly capitalized in documentation.
```

Comment:
s/While/Although/
**while** _conj_
Use “while” only to refer to a period of time. To avoid potential ambiguity, do not use the term as a synonym
for “although” or “though” in documentation.